test/e2e/l7networkpolicy_test.go

```diff
@@ -332,4 +348,64 @@ func testL7NetworkPolicyTLS(t *testing.T, data *TestData) {
 
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "apis.google.com", false)
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "www.facebook.com", true)
+
+	data.crdClient.CrdV1beta1().NetworkPolicies(data.testNamespace).Delete(context.TODO(), policyAllowSNIFacebook, metav1.DeleteOptions{})
+}
+
+func testL7NetworkPolicyLogging(t *testing.T, data *TestData) {
+	l7LoggingNode := nodeName(0)
+	l7LogDir := logDir + "l7engine/eve-" + time.Now().Format(time.DateOnly) + ".json"
+
+	clientPodName := "test-l7-logging-client-selected"
+	clientPodLabels := map[string]string{"test-l7-logging-e2e": "client"}
+	cmd := []string{"bash", "-c", "sleep 3600"}
+	require.NoError(t, NewPodBuilder(clientPodName, data.testNamespace, agnhostImage).OnNode(l7LoggingNode).WithCommand(cmd).WithLabels(clientPodLabels).Create(data))
+	_, err := data.podWaitForIPs(defaultTimeout, clientPodName, data.testNamespace)
+	if err != nil {
+		t.Fatalf("Error when waiting for IP for Pod '%s': %v", clientPodName, err)
+	}
+	require.NoError(t, data.podWaitForRunning(defaultTimeout, clientPodName, data.testNamespace))
+
+	serverPodName := "test-l7-logging-server"
+	serverPodLabels := map[string]string{"test-l7-logging-e2e": "server"}
+	cmd = []string{"bash", "-c", "/agnhost netexec --http-port=8080"}
```

Comment:
you don't need to wrap the `agnhost` command with a bash shell

Reply:
Tested, and updated for tests in the file.